### PR TITLE
Use papercall.io

### PR DESCRIPTION
**Pull request by MarijeP**
https://github.com/scaleconf/scaleconf.github.com/issues/278
and
https://github.com/scaleconf/scaleconf.github.com/issues/279

### CODE REVIEW

File: submit-talk/index.html [hunk 1]
```diff
@@ -60,9 +60,9 @@
         <p>
           There will be a lectern on stage, which can hold a laptop, and
           has a microphone attached. There is also a roaming wireless microphone
-          and a clip-on microphone available. We will have a macbook pro
-          available with keynote installed, but not microsoft powerpoint. We
-          will have connectors available for mac and pc into the a/v system.
+          and a clip-on microphone available. We will have a MacBook Pro
+          available with Keynote installed, but not Microsoft PowerPoint. We
```
Comment (vhata) [inline submit-talk/index.html]: I have Microsoft Powerpoint on my laptop.

File: about/index.html [hunk 2]
```diff
@@ -91,14 +91,9 @@
...
-        <ul class='info-list'>
-          <li>Ben Meier</li>
-          <li>Siobhan O'Donovan</li>
-          <li>Calvin Brizzi</li>
-          <li>Julia Lang</li>
-          <li>Jacques Heunis</li>
-        </ul>
+        <p>Every year the minions help us run the show on the day. Ben, Siobhan,
+          Calvin, Julia, Jacques, Sibulele, Shaheen, Ednecia - thank you!
+        </p>
```
Comment (vhata) [inline about/index.html]: How does this fit into "use papercall.io" ?

File: submit-talk/index.html (hunk 1 above)
Comment (MarijeP) [inline submit-talk/index.html]: It was just a copy paste from the website... I'll update it on both.

File: about/index.html (hunk 2 above)
Comment (MarijeP) [inline about/index.html]: Yeah, sorry, I'd already made the branch and done a commit when I remembered the other changes. Sue me.

**Pull request merged**